wordpress-mobile/WordPress-iOS
# Edit already published images in Gutenberg

Depends on https://github.com/wordpress-mobile/WordPress-iOS/pull/13085

Fixes #13100

This PRs adds the bridge methods to show the Media Editor from Gutenberg.

### To test

1. In `GutenbergViewController.swift` replace the contents of `gutenbergDidRequestFullscreenImage` for this single line: `gutenbergDidRequestMediaEditor(with: mediaUrl, callback: { _ in })` like this:
```swift
    func gutenbergDidRequestFullscreenImage(with mediaUrl: URL) {
        gutenbergDidRequestMediaEditor(with: mediaUrl, callback: { _ in })
    }
```
2. Run the app, open the block editor and double-tap an image
3. The Media Editor should appear

### Additional information

@etoledom This PRs seems big but it's because it depends on another one. The Gutenberg integration is covered on those files:

* `GutenbergViewController.swift`
* `GutenbergMediaInserterHelper.swift`
* `GutenbergMediaEditorImage.swift`

PR submission checklist:
